ceph.spec.in

```diff
@@ -403,7 +402,6 @@ Summary:	OCF-compliant resource agents for Ceph daemons
 %if 0%{?suse_version}
 Group:		System/Filesystems
 %endif
-License:	LGPL-2.0
```

Comment:
yes, a single license would be easier. but if a subpackage can be licensed only under one license (for example, LGPL 2.1), shall reuse the `LGPL-2.1 and CC-BY-SA-1.0 and GPL-2.0 and BSL-1.0 and BSD-3-Clause and MIT` license?